src/libs/Navigation/FreezeWrapper.tsx

```diff
@@ -28,8 +27,8 @@ function FreezeWrapper({keepVisible = false, children}: FreezeWrapperProps) {
             // if the screen is more than 1 screen away from the current screen, freeze it,
             // we don't want to freeze the screen if it's the previous screen because the freeze placeholder
             // would be visible at the beginning of the back animation then
-            if (navigation.getState().index - (screenIndexRef.current ?? 0) > 1) {
-                InteractionManager.runAfterInteractions(() => setIsScreenBlurred(true));
+            if (navigation.getState().index - (screenIndexRef.current ?? 0) >= 1) {
```

Comment:
Does the issue with beginning of animation not occur anymore? And did you try to run it in the new arch? I think the case with `InteractionManager` was needed there only and I think we will want to enable it on web ASAP too.

Reply:
I can pinpoint the any issues with the animations. I haven't tested with new arch, is this even something that is enabled in the app, because it doesn't seem like it.

Reply:
Animation of goBack works ok for me. I haven't tried it with a new arch yet.

Reply:
FYI I also gave it a spin and wasn't able to experience anything weird related to animations.